### DISCUSSION TITLE
Using Intersection Observer, allow lazyload of svg urls (if external url)

### DISCUSSION THREAD

**Discussion by JoshMontech** [gilbarbara/react-inlinesvg#178] [Ideas]
I really enjoy using this package but one thing that's taking away from the experience is that there's no intuitive way to allow lazyloading so my external svg urls coming from my CMS are bloating the page load with SVG url requests. Using something like intersection observer, lazyloading could bring this package to the next level!

**Comment by gilbarbara** [comment]
hey @JoshMontech 

Thanks for your post, but this is not something I would consider adding since this library is optimized to load SVGs...
Adding such functionality would open it up to unnecessary bugs and increase its size considerably.
Also, there are plenty of libraries that can do lazy loading way better.
